doc/build-netbsd.md

```diff
@@ -1,81 +1,161 @@
-NetBSD build guide
-======================
-(updated for NetBSD 8.0)
+# NetBSD Build Guide
 
-This guide describes how to build bitcoind and command-line utilities on NetBSD.
+**Updated for NetBSD [9.1](https://netbsd.org/releases/formal-9/NetBSD-9.1.html)**
 
-This guide does not contain instructions for building the GUI.
+This guide describes how to build bitcoind, command-line utilities, and GUI on NetBSD.
 
-Preparation
--------------
+## Dependencies
 
-You will need the following modules, which can be installed via pkgsrc or pkgin:
+The following dependencies are **required**:
+
+ Library                                          | Purpose    | Description
+ -------------------------------------------------|------------|----------------------
+ [autoconf](https://pkgsrc.se/devel/autoconf)     | Build      | Automatically configure software source code
+ [automake](https://pkgsrc.se/devel/automake)     | Build      | Generate makefile (requires autoconf)
+ [libtool](https://pkgsrc.se/devel/libtool)       | Build      | Shared library support
+ [pkg-config](https://pkgsrc.se/devel/pkg-config) | Build      | Configure compiler and linker flags
+ [git](https://pkgsrc.se/devel/git)               | Clone      | Version control system
+ [gmake](https://pkgsrc.se/devel/gmake)           | Compile    | Generate executables
+ [boost](https://pkgsrc.se/meta-pkgs/boost)       | Utility    | Library for threading, data structures, etc
+ [libevent](https://pkgsrc.se/devel/libevent)     | Networking | OS independent asynchronous networking
+
+
+The following dependencies are **optional**:
+
+  Library                                            | Purpose          | Description
+  ---------------------------------------------------|------------------|----------------------
+  [db4](https://pkgsrc.se/databases/db4)             | Berkeley DB      | Wallet storage (only needed when wallet enabled)
+  [qt5](https://pkgsrc.se/x11/qt5)                   | GUI              | GUI toolkit (only needed when GUI enabled)
+  [qrencode](https://pkgsrc.se/converters/qrencode)  | QR codes in GUI  | Generating QR codes (only needed when GUI enabled)
+  [zeromq](https://pkgsrc.se/net/zeromq)             | ZMQ notification | Allows generating ZMQ notifications (requires ZMQ version >= 4.0.0)
+  [sqlite3](https://pkgsrc.se/databases/sqlite3)     | SQLite DB        | Wallet storage (only needed when wallet enabled)
+  [python37](https://pkgsrc.se/lang/python37)        | Testing          | Python Interpreter (only needed when running the test suite)
```

Comment:
According to doc/dependencies.md, the minimum required Python version is currently 3.6; if we can get away with specifying Python 3.6 and up, that might allow this doc to stay current longer.

Reply:
Good

Reply:
lets leave this like this for now, when the minimum supported is 3.8, we can update it. What do you think?

Should be a while before the minimum is 3.8, right?

Reply:
I don't have a strong opinion.